build.gradle

```diff
@@ -121,7 +121,7 @@ jar {
   duplicatesStrategy = 'exclude'
   destinationDirectory = file('dist/')
   archiveBaseName = "KoLmafia"
-  classifier = "${versioning.info.dirty ? 'Modified' : ''}"
+  classifier = "${versioning.info.dirty ? 'M' : ''}"
```

Comment:
(Note this is subtly different from what we have today with `ant daily`, namely we'll now have "KoLmafia-r12345-M.jar" instead of "KoLmafia-r12345M.jar". I don't think this is a meaningful difference.)

Reply:
Yeah me and @BadHorseMonkey noticed this and thought that it might be alright given it allows you to split by - and grab [1]